db/data_migration/20190222130820_reslug_documents.rb

```diff
@@ -0,0 +1,56 @@
+puts "Reslugging documents..."
+
+documents = [
+  {
+    slug: "letter-to-providers-on-health-and-safety",
+    new_slug: "letters-from-the-regulator-of-social-housing-Grenfell",
```

Comment:
shall we lowercase this - since as you say it'll do it automatically? at least makes our expectations clearer.